src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.ui.Ui;
+import duke.util.TaskList;
+
+/**
+ * Represents a command that deletes the specified task from the task list.
+ */
+public class DeleteCommand extends Command {
+    private int index;
+
+    public DeleteCommand(int index) {
+        this.index = index;
+    }
+
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        Task t = tasks.getTask(index);
```

Comment:
Would it be clearer if `t` was renamed as `task`?

Reply:
Hi, thanks for the feedback, I've renamed `t` as `deletedTask`.